`make install-capi-lib` and `make install-pkgconfig` fail unless wasmer binary is built.

### Describe the bug

Using Rust 1.54 on CloudLinux 6 and a checked-out version of wasmer’s git repo:

- `make build-capi` works fine
- `make install-capi-headers` works

… but that’s where success stops:
```
# make install-capi-lib
-----------
INFORMATION
-----------

/bin/bash: rustup: command not found
Host Target: ``.
Enabled Compilers: cranelift, singlepass.
Testing the following compilers & engines:
* API: cranelift-universal cranelift-dylib singlepass-universal
* C-API: cranelift-universal cranelift-dylib
Cargo features:
* Compilers: `--features cranelift,singlepass`.


--------------
RULE EXECUTION
--------------


pkgver=$(target/release/wasmer --version | cut -d\  -f2) && \
	shortver="${pkgver%.*}" && \
	majorver="${shortver%.*}" && \
	install -Dm755 target/release/libwasmer.so "/usr/local/lib/libwasmer.so.$pkgver" && \
	ln -sf "libwasmer.so.$pkgver" "/usr/local/lib/libwasmer.so.$shortver" && \
	ln -sf "libwasmer.so.$pkgver" "/usr/local/lib/libwasmer.so.$majorver" && \
	ln -sf "libwasmer.so.$pkgver" "/usr/local/lib/libwasmer.so"
/bin/bash: target/release/wasmer: No such file or directory
install: accessing `/usr/local/lib/libwasmer.so.': Too many levels of symbolic links
make: *** [install-capi-lib] Error 1
```
… and:
```
# make install-pkgconfig
-----------
INFORMATION
-----------

/bin/bash: rustup: command not found
Host Target: ``.
Enabled Compilers: cranelift, singlepass.
Testing the following compilers & engines:
* API: cranelift-universal cranelift-dylib singlepass-universal
* C-API: cranelift-universal cranelift-dylib
Cargo features:
* Compilers: `--features cranelift,singlepass`.


--------------
RULE EXECUTION
--------------


# Make sure WASMER_INSTALL_PREFIX is set during build
unset WASMER_DIR; \
	if pc="$(target/release/wasmer config --pkg-config 1>/dev/null 2>/dev/null)"; then \
		echo "$pc" | install -Dm644 /dev/stdin "/usr/local"/lib/pkgconfig/wasmer.pc; \
	else \
		echo 1>&2 "WASMER_INSTALL_PREFIX was not set during build, not installing wasmer.pc"; \
	fi
WASMER_INSTALL_PREFIX was not set during build, not installing wasmer.pc
```

### Steps to reproduce

See above.

### Expected behavior

Since the Makefile exposes the library-only build targets, the library-only installation targets should work in isolation; currently they require the binary to be built.

### Actual behavior

Failure as shown above.

### Additional context

CloudLinux 6’s gcc is too old to build the binary; there’s a dependency that expects `-std=c11` to work, but it doesn’t in this older gcc. The library builds, though, without issue.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -17,6 +17,7 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 - [#2942](https://github.com/wasmerio/wasmer/pull/2942) Fix clippy lints.
 - [#2943](https://github.com/wasmerio/wasmer/pull/2943) Fix build error on some archs by using c_char instead of i8
 - [2976](https://github.com/wasmerio/wasmer/pull/2976) Upgrade minimum enumset to one that compiles
+- [2988](https://github.com/wasmerio/wasmer/pull/2988) Have make targets install-capi-lib,install-pkgconfig work without building the wasmer binary
 
 ## 2.3.0 - 2022/06/06
 

--- a/Makefile
+++ b/Makefile
@@ -648,6 +648,8 @@ DESTDIR ?= /usr/local
 
 install: install-wasmer install-capi-headers install-capi-lib install-pkgconfig install-misc
 
+install-capi: install-capi-headers install-capi-lib install-capi-pkgconfig install-misc
+
 install-wasmer:
 	install -Dm755 target/release/wasmer $(DESTDIR)/bin/wasmer
 
@@ -657,7 +659,7 @@ install-capi-headers:
 
 # Currently implemented for linux only. TODO
 install-capi-lib:
-	pkgver=$$(target/release/wasmer --version | cut -d\  -f2) && \
+	pkgver=$$($(CARGO_BINARY) pkgid --manifest-path lib/c-api/Cargo.toml | sed --posix 's/^.*wasmer-c-api:\([0-9.]*\)$\/\1/') && \
 	shortver="$${pkgver%.*}" && \
 	majorver="$${shortver%.*}" && \
 	install -Dm755 target/release/libwasmer.so "$(DESTDIR)/lib/libwasmer.so.$$pkgver" && \
@@ -667,6 +669,10 @@ install-capi-lib:
 
 install-misc:
 	install -Dm644 LICENSE "$(DESTDIR)"/share/licenses/wasmer/LICENSE
+
+install-capi-pkgconfig:
+	@pkgver=$$($(CARGO_BINARY) pkgid --manifest-path lib/c-api/Cargo.toml | sed --posix 's/^.*wasmer-c-api:\([0-9.]*\)$\/\1/') && \
+	printf "prefix=%s\nincludedir=\044{prefix}/include\nlibdir=\044{prefix}/lib\n\nName: wasmer\nDescription: The Wasmer library for running WebAssembly\nVersion: %s\nCflags: -I\044{prefix}/include\nLibs: -L\044{prefix}/lib -lwasmer\n" "$(DESTDIR)" "$${pkgver}" | install -Dm644 /dev/stdin "$(DESTDIR)"/lib/pkgconfig/wasmer.pc
 
 install-pkgconfig:
 	# Make sure WASMER_INSTALL_PREFIX is set during build

